libconnman-qt/commondbustypes.cpp

```diff
@@ -27,3 +27,17 @@ const QDBusArgument &operator>>(const QDBusArgument &argument, ConnmanObject &ob
     argument.endStructure();
     return argument;
 }
+
+void registerCommonDataTypes()
+{
+    static bool initialized = false;
+    if (!initialized) {
+        qDBusRegisterMetaType<StringMap>();
+        qDBusRegisterMetaType<StringPair>();
+        qDBusRegisterMetaType<StringPairArray>();
+        qDBusRegisterMetaType<ConnmanObject>();
+        qDBusRegisterMetaType<ConnmanObjectList>();
+        qRegisterMetaType<ConnmanObjectList>("ConnmanObjectList");
+        initialized = true;
```

Comment:
nit: as there is no reason not to se the flag, I'd do it first

Reply:
Sure, i'll update.